presto-docs/src/main/sphinx/release/release-0.179.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Release 0.179
+=============
+
+General Changes
+---------------
+
+* Fix issue which could cause incorrect results when processing dictionary encoded data. If the expression
+  can fail on bad input, the results from filtered-out rows containing bad input may be included the query
+  output. See `#8262 <https://github.com/prestodb/presto/issues/8262>`_ for more details.
+* Fix planning failure when similar expressions appear in the ``ORDER BY`` clause of a query that
+  contains ``ORDER BY`` and ``LIMIT``.
+* Fix planning failure when ``GROUPING()`` is used while ``legacy_order_by`` session property is ``true``.
+* Fix parsing failure when ``NFD``, ``NFC``, ``NFKD`` or ``NFKC`` are used as identifiers.
+* Fix a memory leak on the coordinator that manifests itself with canceled queries.
+* Fix excessive GC overhead caused by captured lambda expressions.
+* Reduce memory usage for map/array aggregation functions.
+* Redact sensitive config property values in the server log.
+* Updated timezone rule to version 2017b.
+* Add :func:`repeat` function to repeat an element for multiple times.
+* Add :func:`crc32` function.
+* Add file based global security, which can be configured with the ``etc/access-control.properties``
+  and ``security.config-file`` config properties. See :doc:`/security/built-in-system-access-control`
+  for more details.
+* Add query queued and running time limits to resource groups.
```

Comment:
Add support for configuring query runtime and queueing time limits to resource groups.